dist/gh-emoji.js

```diff
@@ -51,14 +51,26 @@
   }
 
   function parse(text) {
+    var options = arguments.length <= 1 || arguments[1] === undefined ? {} : arguments[1];
+
     var output = '';
     output += text.replace(delimiterRegex, function (match) {
-      var id = match.replace(/:/g, '');
-      if (exist(id)) {
-        var classNames = 'gh-emoji gh-emoji-' + id;
-        return '<img src="' + getUrl(id) + '" class="' + classNames + '" alt="' + id + '" />';
+      var name = match.replace(/:/g, '');
+      var classNames = ['gh-emoji', 'gh-emoji-' + name];
+
+      if (!exist(name)) {
+        return match;
+      }
+
+      if (options.classNames) {
+        [].push.apply(classNames, options.classNames.trim().split(/\s+/));
       }
-      return match;
+
+      var imageSrc = getUrl(name);
+      var imageClass = classNames.join(' ');
+      var imageAlt = name;
+
+      return '<img src="' + imageSrc + '" class="' + imageClass + '" alt="' + imageAlt + '" />';
```

Comment:
Doesn't actually enforce the airbnb eslint config template-literals?

With template-literals, you could write this:
```
return `<img src="${imageSrc}" class="${imageClass}" alt="${imageAlt}" />`;
```

Which looks much easier on the eyes :-)
What you think?